recipe/magento2.php

```diff
@@ -23,10 +24,34 @@
 
 // You can also set the themes to run against. By default it'll deploy
 // all themes - `add('magento_themes', ['Magento/luma', 'Magento/backend']);`
+// If the themes are set as a simple list of strings, then all languages defined in {{static_content_locales}} are
+// compiled for the given themes.
+// Alternatively The themes can be defined as an associative array, where the key represents the theme name and
+// the key contains the languages for the compilation (for this specific theme)
+// Example:
+// set('magento_themes', ['Magento/luma']); - Will compile this theme with every language from {{static_content_locales}}
+// set('magento_themes', [
+//     'Magento/luma'   => null,                              - Will compile all languages from {{static_content_locales}} for Magento/luma
+//     'Custom/theme'   => 'en_US fr_FR'                      - Will compile only en_US and fr_FR for Custom/theme
+//     'Custom/another' => '{{static_content_locales}} it_IT' - Will compile all languages from {{static_content_locales}} + it_IT for Custom/another
+// ]); - Will compile this theme with every language
 set('magento_themes', [
 
 ]);
 
+// Static content deployment options, e.g. '--no-parent'
+set('static_deploy_options', '');
+
+// Deploy frontend and adminhtml together as default
+set('split_static_deployment', false);
+
+// Use the default languages for the backend as default
+set('static_content_locales_backend', '{{static_content_locales}}');
+
+// backend themes to deploy. Only used if split_static_deployment=true
+// This setting supports the same options/structure as {{magento_themes}}
+set('magento_themes_backend', ['Magento/backend']);
```

Comment:
the default value should be `['Magento/backend' => null]`

Reply:
you are right. I'll fix this later